src/js/subtraction/components/Selector.js

```diff
@@ -1,28 +1,121 @@
-import { map } from "lodash-es";
 import PropTypes from "prop-types";
 import React from "react";
-import { MultiSelector, MultiSelectorItem } from "../../base/MultiSelector";
+import { MultiSelector } from "../../base/MultiSelector";
 import { Link } from "react-router-dom";
 import { NoneFoundBox } from "../../base/NoneFound";
+import styled from "styled-components";
+import { BoxGroupSection, Icon, Input } from "../../base";
+import { AllSelectedBox, SelectorItem } from "../../analyses/components/Create/WorkflowSelector";
 
-export const SampleSubtractionSelector = ({ name, noun, selected, subtractions, onChange }) => {
-    const subtractionComponents = map(subtractions, ({ name, id }) => (
-        <MultiSelectorItem key={id} name={name} value={id} id={id}>
-            <span>{name}</span>
-        </MultiSelectorItem>
-    ));
-    if (!subtractions.length) {
+export const SampleInputContainer = styled.div`
```

Comment:
```suggestion
export const SearchbarContainer = styled.div`
```
(This change assumes `SampleInputContainer`  really does only get used as a SearchBar. Sample should probably be dropped from the name either way as I don't think there is anything sample specific about this component)